README.md

```diff
@@ -8,32 +8,83 @@ The comparison happens offline.
 
 Special thanks to Troy Hunt ([@troyhunt](https://twitter.com/troyhunt)) for making this script possible.
 
-## install
+## Installation
 
 ``` bash
 pip install pywnedpasswords
 ```
 
 ## Usage
 
+### Interactive 
+
 ``` bash
-pywnedpasswords.py
+pywnedpasswords
 ```
 
 Insert your password when asked.
 
 the output will either be:
 
-```
-Password to check:
-Found your password 47205 times.
-```
+> Password to check:
+> 
+> Found your password 47205 times.
 
 or in case your password is not secure
 
+> Password to check:
+> 
+> Your password did not appear in PwnedPasswords yet.
+
+
+### Passing the password as a command line argument
+
+
+``` bash
+pywnedpasswords Passw0rd
```

Comment:
while it makes sense to support this - I think this should be marked as "discouraged" - as it leaves the password in the (bash) history.